pkgs/tools/inputmethods/gebaar-libinput/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, cmake, libinput, zlib }:
+stdenv.mkDerivation rec {
+  pname = "gebaar-libinput";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Coffee2CodeNL";
+    repo = "gebaar-libinput";
+    rev = "v${version}";
+    sha256 = "1kqcgwkia1p195xr082838dvj1gqif9d63i8a52jb0lc32zzizh6";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
+  buildInputs = [ libinput zlib ];
+
+  meta = with lib; {
+    description =
```

Comment:
Put it in one line, this isn't the `longDescription` attribute.

Reply:
It is one line, just with a line break between = and "; Again, nixfmt does this I just leave it alone.